src/components/HomePage/Banner.js

```diff
@@ -40,7 +42,11 @@ const useStyles = makeStyles(theme => ({
 	timer: {
 		color: 'white',
 		fontWeight: 600,
-		fontSize: '4em'
+		fontSize: '4em',
+		textAlign: 'center',
+		[theme.breakpoints.down('xs')]: {
```

Comment:
I think the addition of the rule

```js
[theme.breakpoints.down('sm')]: {
  fontSize: '3em'
}
```

would work even better than this change alone. Right now at the `sm` size, the timer still appears bigger than the "Hack on the Hill 7" text itself, which looks a bit strange.